.github/workflows/create-manifest-pr.yaml

```
@@ -0,0 +1,16 @@
+name: Create manifest PR
+
+on:
+  pull_request_target:
+    types: [opened, synchronize, closed]
+    branches:
+      - main
```

```suggestion
      - master
```
We use master branch in Matter.